src/platform/modulesmanager/tests/inc/MockManagementModule.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef MOCKMANAGEMENTMODULE_H
+#define MOCKMANAGEMENTMODULE_H
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <ManagementModule.h>
+
+namespace Tests
+{
+    class MockManagementModule : public ManagementModule
+    {
+    public:
+        MockManagementModule(const std::string& clientName, unsigned int maxPayloadSizeBytes = 0);
+
+        MOCK_METHOD(void, LoadModule, ());
+        MOCK_METHOD(void, UnloadModule, ());
+        MOCK_METHOD(int, CallMmiSet, (const char* componentName, const char* objectName, const MMI_JSON_STRING payload, const int payloadSizeBytes));
+        MOCK_METHOD(int, CallMmiGet, (const char* componentName, const char* objectName, MMI_JSON_STRING* payload, int* payloadSizeBytes));
+
+        void MmiSet(Mmi_Set mmiSet);
+        void MmiGet(Mmi_Get mmiGet);
```

Comment:
No MmiGetInfo, MmiOpen, MmiClose, MmiFree?

Reply:
`MmiGetInfo`, `MmiOpen`, `MmiClose`, and `MmiFree`are exercised in the constructor of each `ManagmentModule` by the test modules.

Reply:
With some changes to the `ManagementModule` constructor I can add tests to exercise the validation of `MmiGetInfo()`

Reply:
I will make this modification

Reply:
Would be better to have test wrappers for each individually at the same level, even if some will be invoked by other tests.